go/client/cmd_prove.go

```diff
@@ -104,7 +104,7 @@ func (p *CmdProve) installOutputHook(ui *ProveUI) {
 
 // NewCmdProve makes a new prove command from the given CLI parameters.
 func NewCmdProve(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
-	serviceList := strings.Join(g.Services.ListProofCheckers(), ", ")
+	serviceList := strings.Join(g.Services.ListProofCheckers(g.GetRunMode()), ", ")
```

Comment:
This call to `g.GetRunMode()` doesn't seem to actually respect the `--run-mode` flag. I need to dig into it.